src/App.jsx

```diff
@@ -134,6 +135,94 @@ function App() {
                 each year without running out of money for at least 30 years. This calculation 
                 assumes your investments continue to grow during retirement.
               </p>
+              <button 
+                className="learn-more-btn"
+                onClick={() => setShowRuleModal(true)}
+              >
+                Learn More About the 4% Rule
```

Comment:
🔰 **Junior Developer**

Wow, I really like the new modal UI! The CSS looks super clean and the gradients are nice! 😍

I noticed in `App.jsx` on line 142, the button text says "Learn More About the 4% Rule". Should we maybe make it more consistent with the modal header (which is "The 4% Rule: A Comprehensive Guide")? Or is there a reason for the difference? Just curious!

Also, is it better to use `aria-modal="true"` for accessibility on the modal overlay? Or maybe add `role="dialog"` too? Sorry, I'm still learning about accessibility stuff 😅